client/connector.go

```diff
@@ -109,12 +109,15 @@ func (api *APIClient) DeleteConnectorToken(ctx context.Context, connectorID, tok
 // Connector represents a connector in your Border0 organization.
 type Connector struct {
 	// input and output fields
-	Name                     string `json:"name"`
-	Description              string `json:"description,omitempty"`
-	BuiltInSshServiceEnabled bool   `json:"built_in_ssh_service_enabled,omitempty"`
+	Name        string `json:"name"`
+	Description string `json:"description,omitempty"`
 
 	// output field
 	ConnectorID string `json:"connector_id"`
+
+	// built-in SSH service
+	BuiltInSshServiceEnabled bool    `json:"built_in_ssh_service_enabled,omitempty"`
+	BuiltInSshService        *Socket `json:"built_in_ssh_service,omitempty"` // optional, nil if built-in SSH service is disabled
```

Comment:
Can we return just the ID?

Reply:
I could do that in terraform, just return the ssh socket id without the nested object. In SDK, it deserializes json returned from api, so the nested socket object is needed.